src/Components/Modal.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+export default function Modal({ title, children, dismissable, onclose }) {
```

Comment:
Please take a look at onclose variable here, you can rename it as `onClose`

Reply:
I have renamed it to onClose.
```
export default function Modal({ title, children, dismissable, onClose }) {
```